src/components/ContentNode/LinkableHeading.vue

```diff
@@ -41,9 +43,10 @@ export default {
       type: String,
       required: false,
     },
-    tag: {
-      type: String,
-      default: () => 'h2',
+    level: {
+      type: Number,
+      default: () => 2,
+      validator: v => v in HeadingLevels,
```

Comment:
I don't think the `in` operator works like you're expecting it to here.

Example:
```javascript
> 0 in [1, 2, 3, 4, 5, 6]
true
> 6 in [1, 2, 3, 4, 5, 6]
false
```

It might be better to just check if the number is in bounds. Something like:
```javascript
v => v >= 1 && v <= 6
```

Reply:
Oh, thank you! it seemed I was using it wrong, yeah :D fixed it!